products section html markup

ViacheslavKozlov opened the pull request.

Commit:
products section html markup

AlexMoskalyk commented on src/partials/products.html:
```diff
@@ -0,0 +1,75 @@
+<section class="container">
```
для контейнера сделать отдельный div, прописать класы

ViacheslavKozlov replied:
добавил div для container
добавил классы
закомментировал "точки" до уточнения с Андреем как их делать

Commit:
updated html markup

- added classes,
- added div for container class
- commented "dots" for further instructions from mentor.

AlexMoskalyk commented on src/partials/products.html:
```diff
@@ -0,0 +1,77 @@
+<section>
```
Добавить класс секции, в следующей ветке,
